app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivityTest.java

```diff
@@ -170,4 +179,17 @@ public void whenAllNecessaryPermissionsAreGranted_prepareViewIsCalled() {
         //Then
         verify(spyActivity).prepareView();
     }
+
+    private EditActivity createActivity() {
```

Comment:
You can create activity in setup() method. It will be created before each test starts.